FabLabKasse/faucardPayment/FAUcardPaymentThread.py

```diff
@@ -0,0 +1,554 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+#
+
+import codecs
+import logging
+import sqlite3
+from PyQt4 import QtCore, Qt
+from decimal import Decimal
+from datetime import datetime
+
+from faucardStates import Status, Info
+from MagPosLog import MagPosLog
+
+from magpos import magpos, codes
+
+
+import scriptHelper
+from ConfigParser import ConfigParser
+
+cfg = scriptHelper.getConfig()
+cfg.readfp(codecs.open('config.ini', 'r', 'utf8'))
```

Comment:
Line 22 seems to be a fragment from older states which I forgot to remove from the code. I will delete it in the next commit.